### PR TITLE
Wagtail 6.0

### PR DESCRIPTION
**Pull request by katdom13**
[Support Ticket](https://torchbox.monday.com/boards/1124794299/pulses/1396934053)

### Summary of changes

- Adjust test matrix to use Wagtail 6.0
- Drop Wagtail < 5.2 from the test matrix
- Add Python 3.12 to test matrix
- Update minimum Wagtail version to 5.2 in setup.py

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [tox]
 minversion = 3.14.6
 envlist =
-    py{38,39,310}-dj{32,41}-wagtail{41,51,52}
-    py311-dj{42}-wagtail{51,52}
+    py{38,39,310}-dj32-wagtail52
+    py{38,39,310,311}-dj42-wagtail{52,60}
+    py{311,312}-dj50-wagtail{52,60}
```
Comment (Stormheg) [inline tox.ini]: Technically, Python 3.10 is also supported by Django 5.0. https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support

```suggestion
    py{310,311,312}-dj50-wagtail{52,60}
```

**Review by Stormheg — APPROVED** [review]
Looks good to me!

@katdom13 I notice we still have the Python 3.7 trove classifier in `setup.py`, could you please remove that?

File: tox.ini (hunk 1 above)
Comment (katdom13) [reply tox.ini]: Thanks @Stormheg , applied this one!

**Review by Stormheg — APPROVED** [review]
Brilliant!

**Pull request merged**